index.css

```diff
@@ -127,6 +127,22 @@ nav {
 
 .menu-links li a {
   text-decoration: none;
+  cursor: pointer;
+}
+
+.menu-links li a:hover {
+  background-color: #7f8cff;
+  padding: 8px;
+  color: #fff;
+  border-radius: 4px;
+}
```

Comment:
For most websites, padding should be in the initial state 
`.menu-links li a { padding: 8px }`

If you take from the design template provided, it will be more appropriate to give underline when hover.

This is not very important, it's just my suggestion